boards/arm/nrf5340dk_nrf5340/nrf5340dk_nrf5340_cpuappns.yaml

```diff
@@ -6,7 +6,7 @@ toolchain:
   - gnuarmemb
   - xtools
   - zephyr
-ram: 384
+ram: 192
```

Comment:
@ioannisg Should not [nrf5340dk_nrf5340_cpuapp.yaml](https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf5340dk_nrf5340/nrf5340dk_nrf5340_cpuapp.yaml#L9) also be adapted?

Reply:
Not really. That one should reflect _all_ memory that is available when we do not consider S, and NS builds. And this is already the case.